### PR TITLE
:book: v0.5.0 docs

### PR DESCRIPTION
**Pull request by detiber**
**What this PR does / why we need it**:

Updates docs for v0.5.0
Updates available AMIs

/milestone v0.5.0

### CODE REVIEW

**Comment by detiber** [comment]
I suspect this is probably good enough to start, I filed an issue #1638 for updating the docs around accessing instances, since it was a bit more than I wanted to try to bite off today.

/assign @ncdc

File: README.md [hunk 1]
```diff
@@ -42,11 +42,12 @@ cluster on AWS.
...
-|-|-|-|
-|AWS Provider v1alpha1 (v0.2)|✓||
-|AWS Provider v1alpha1 (v0.3)|✓||
-|AWS Provider v1alpha2 (v0.4)||✓|
+||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (v0.2)|Cluster API v1alpha3 (v0.3)|
+|-|-|-|-|
+|AWS Provider v1alpha1 (v0.2)|✓|||
+|AWS Provider v1alpha1 (v0.3)|✓|||
+|AWS Provider v1alpha2 (v0.4)||✓||
+|AWS Provider v1alpha2 (v0.5)|||✓|
```
Comment (ncdc) [inline README.md]:
```suggestion
|AWS Provider v1alpha3 (v0.5)|||✓|
```

File: README.md [hunk 2]
```diff
@@ -55,6 +56,7 @@ This provider's versions are able to install and manage the following versions o
 |AWS Provider v1alpha1 (v0.2)|✓|✓|✓|||
 |AWS Provider v1alpha1 (v0.3)|✓|✓|✓|||
 |AWS Provider v1alpha2 (v0.4)||✓|✓|✓|✓|
+|AWS Provider v1alpha2 (v0.5)|||✓|✓|✓|
```
Comment (ncdc) [inline README.md]:
```suggestion
|AWS Provider v1alpha3 (v0.5)|||✓|✓|✓|
```

File: README.md (hunk 1 above)
Comment (detiber) [reply README.md]: :grimacing: fixed in latest

File: README.md (hunk 2 above)
Comment (detiber) [reply README.md]: :grimacing: fixed in latest

**Comment by ncdc** [comment]
/lgtm
thanks!

**Pull request merged**